include/nil/crypto3/zk/snark/systems/plonk/pickles/proof.hpp

```diff
@@ -40,6 +40,8 @@ namespace nil {
                 struct lookup_st {
                     std::vector<CommitmentType> sorted;
                     CommitmentType aggreg;
+                    CommitmentType table;
```

Comment:
I think, we need to rename template parameter here because we use the same structure with a finite field.